README.md

```diff
@@ -0,0 +1,128 @@
+<head>
+  <meta name="Momento Laravel cache driver example" content="Taggable Momento serverless cache driver example for Laravel">
+</head>
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+# Momento Laravel Example
+In this repo, you will see an example of how to integrate a Momento cache into your Laravel app, instead of a Redis or Memcached.
+What's the benefits, you ask? You don't have to worry about Redis/Memcached nodes!
+Intrigued? Keep on reading!
+
+## Run the app via Docker
+
+Build a Docker image for the app:
+
+Coming soon.
+
+## Manual PHP Setup
+
+Need to install the following:
+
+- [PHP](https://www.php.net/manual/en/install.macosx.packages.php)
+- [Composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-macos)
+- [gRPC for PHP](https://cloud.google.com/php/grpc)
+
+Add the repository and dependency to your project's `composer.json`:
+
+```json
+{
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/momentohq/laravel-example"
+    },
+    {
+      "type": "vcs",
+      "url": "https://github.com/momentohq/laravel-cache"
+    },
+    {
+      "type": "vcs",
+      "url": "https://github.com/momentohq/client-sdk-php"
+    }
+  ],
+  "require": {
+    "momentohq/laravel-example": "0.1.0",
+    "momentohq/laravel-cache": "0.1.2",
+    "momentohq/client-sdk-php": "0.2.0"
```

Comment:
```suggestion
    "momentohq/client-sdk-php": "0.2.1"
```

Reply:
We also need to update the version requirement for `momentohq/client-sdk-php` in the `laravel-cache` repository.